challenges/2/YOUR-CODE-GOES-HERE/reihanehPourkaka/challenge/src/App.vue

```diff
@@ -0,0 +1,13 @@
+<script setup>
+import Clock from './components/Clock.vue'
+
+
+
+</script>
+
+<template>
+  <div class="container">
+    <Clock :activeHours='[{start: 5, end: 13}, {start: 22, end: 24}]'/>
```

Comment:
When I change these values (5, 13, 22, 24) component does not re-render.
Can you say why? and fix it?

Reply:
hi
Yes , i forgot add watch .
i fixed it.